scripts/base/init-bare.zeek

```diff
@@ -380,6 +380,10 @@ export {
 		send_timestamp: count &optional;
 		## Kind 8: 4-byte echo reply timestamp value.
 		echo_timestamp: count &optional;
+		## Kind 27: TCP Quick Start Response value.
+                rate: count &optional;
+                ttl_diff: count &optional;
+                qs_nonce: count &optional;
```

Comment:
Looks like the whitespace is a bit off here.

Reply:
Sorry about that, copy paste from the server to the github sometime messes with the whitespace. Fixed. :)